### PR TITLE
remove the css from the html

### PR DESCRIPTION
**Pull request by chensokheng123**
What do you think?

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+  padding:0px ;
+  margin: 0px;
+}
+body
+{
+  display: flex;
+  flex-wrap: wrap;
+}
+
+body {
```
Comment (abhieshekumar) [inline css/style.css]: Mistakenly, two `body` selectors were declared previously, can you combine the rules into a single `body` selector.

File: index.html [hunk 2]
```diff
@@ -1,187 +1,70 @@
...
     </div>
   </div>
-</div>
-<div id="foot">
+  <div id="foot">
 
   </div>
 </body>
+
 </html>
```
Comment (daemonsloth) [inline index.html:69]: maybe add a new line here..

**Review by daemonsloth — DISMISSED** [review]

**Review by abhieshekumar — CHANGES_REQUESTED** [review]

File: css/style.css (hunk 1 above)
Comment (chensokheng123) [reply css/style.css]: okay, I am on it.

**Comment by chensokheng123** [comment]
I already have done what you want to me do.

**Review by abhieshekumar — APPROVED** [review]

**Comment by daemonsloth** [comment]
@Chensokheng new line at the end of the files ? , in future we will put linter for these things, but for now let's do manually

File: index.html (hunk 2 above)
Comment (chensokheng123) [reply index.html:69]: Hi there, Can I know what line should I add? You mean add a line of text to the footer?

Comment (chensokheng123) [reply index.html:69]: I am not clear about it.

Comment (savvysiddharth) [reply index.html]: I meant just a new line at the end of file

Comment (daemonsloth) [reply index.html:69]: I meant just a new line at the end of file, after the last line of code

Comment (daemonsloth) [reply index.html:69]: Just a standard to follow, more about it https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Comment (chensokheng123) [reply index.html:69]: well, I am so sorry. but I still don't get it. You mean to add a new line. Is it a line of what?

Comment (daemonsloth) [reply index.html:69]: Umm..leave it, we will do it later,.. will merge ur PR

**Comment by daemonsloth** [comment]
Thanks @Chensokheng

**Comment by chensokheng123** [comment]
My pleasure to do this. Thank you

**Pull request merged**